soh/src/overlays/actors/ovl_En_Mb/z_en_mb.c

```diff
@@ -1235,10 +1260,21 @@ void EnMb_ClubWaitPlayerNear(EnMb* this, PlayState* play) {
     s32 pad;
     s16 relYawFromPlayer = this->actor.world.rot.y - this->actor.yawTowardsPlayer;
 
+    // Rotate Club Moblin towards player in Enemy Randomizer because they're borderline
+    // useless otherwise in most scenarios.
```

Comment:
```suggestion
    // Rotate Club Moblin towards player in Enemy Randomizer because they're
    // borderline useless otherwise in most scenarios.
```